pages/common/llm.md

```diff
@@ -19,10 +19,10 @@
 
 `llm install {{package1 package2 ...}}`
 
-- Download and run a prompt against a [m]odel:
+- Download and run a prompt against a [model]:
```

Comment:
```suggestion
- Download and run a prompt against a model:
```

I am sorry for the confusion caused earlier. I went through the [Style Guide Short Option Mnemonics](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#short-option-mnemonics] again; complete words options like `model` should not have any mnemonic at all. @kbdharun or @vitorhcl can clarify here.

Reply:
Exactly, mnemonics only be used to highlight short options or their context.